sycl/test/basic_tests/info.cpp

```diff
@@ -14,27 +14,41 @@
 #include <CL/sycl.hpp>
 #include <iostream>
 #include <string>
+#include <strstream>
 
 using namespace cl::sycl;
 
-template <typename T> std::string info_to_string(T info) {
+// Handle unknown info, e.g., from non-standard extensions.
+template <typename T>
+std::string unknown_info_to_string(T info) {
+  std::strstream stream;
+  stream << std::hex << static_cast<unsigned long>(info) << " (unknown value)"
```

Comment:
I have a question regarding the type 'unsigned long' chosen on this line (i.e. 32-bit type on Windows64 and 64-bit on Linux64).
Do you have some special reasoning to choose 'unsigned long' instead of some 64-bit type here?

This function may be called with 'info' which has type 'info::device_type' where: 'device_type' seems to be based on 64-bit type:
enum class device_type : pi_uint64 {

Reply:
Thank you for catching this! Will change it based on your suggestion.

Reply:
Pushed.